pkg/prowgen/prowgen.go

```diff
@@ -137,9 +139,9 @@ func testContainsLease(test *cioperatorapi.TestStepConfiguration) bool {
 	return len(api.LeasesForTest(test.MultiStageTestConfigurationLiteral)) > 0
 }
 
-func generatePresubmitForTest(jobBaseBuilder *prowJobBaseBuilder, name string, info *ProwgenInfo, runIfChanged, skipIfOnlyChanged string, optional bool) *prowconfig.Presubmit {
+func generatePresubmitForTest(jobBaseBuilder *prowJobBaseBuilder, name string, info *ProwgenInfo, runIfChanged, skipIfOnlyChanged string, optional, disableRehearsal bool) *prowconfig.Presubmit {
```

Comment:
Can we instead add an optional argument that holds any options including the rehearse boolean? This way you don't have to update all the references of this function, and also allows the code to be more extensible in the future if we want to add more options.